### PR TITLE
Added Short video Delete Button

### PR DESCRIPTION
**Pull request by moezthedev**
# Pull Request Template

Before submitting this PR, please make sure you have completed the following checklist. This helps ensure that our codebase remains high quality and that our features work well across all supported devices and themes.

### Code Review
- [x] I have carefully reviewed my code and believe it is final from my side.
- [x] I have ensured there is no hardcoded color in my changes. Colors should be defined in theme files and reused.

### Theme Compatibility
- [x] I have tested the new feature or updates in both light and dark themes.
- [x] I confirm that the visual aspects of the feature are compatible and appealing in both themes.

### Device Compatibility
- [x] I have tested my changes on mobile screen sizes.
- [x] I have tested my changes on tablet screen sizes.
- [ ] I have tested my changes on laptop screen sizes.

### Functional Testing
- [x] I have thoroughly tested the functionality of my changes.
- [x] I am confident that my changes do not negatively impact existing features.

### Testing Documentation
Please provide a brief description of the tests you have performed to verify your changes. Include any relevant scenarios or edge cases.

- Example test 1: Tested feature X by uploading the image
- Example test 2: Tested feature X by doing these things.

# Demo Video
Please provide a demo video of your changes. This helps reviewers understand the changes and verify the functionality.

You can record the video using tools like Loom(https://www.loom.com/), Clip by ClickUp(https://clickup.com/features/clips), Sendspark(https://www.sendspark.com/), etc.

Include the link to the video below and make sure the video covers
- [x] Explain the task/bug you are solving
- [x] Show the changes in the UI
- [x] Show the changes in the codebase
- [x] Show the changes you tested to make sure it works
- [x] Show the changes in various screen sizes
Video: https://www.loom.com/share/80a0f23b7387449da76c9745257f8d02

### CODE REVIEW

File: academy-ui/src/components/shortVideos/View/Shorts.tsx [hunk 1]
```diff
@@ -1,32 +1,64 @@
...
-          <h1>{title}</h1>
+    <div className="flex-row justify-between items-start relative ">
+      {showDeleteModal && (
+        <DeleteConfirmationModal
+          title={'Delete Short Video'}
+          open={showDeleteModal}
+          onClose={() => setShowDeleteModal(false)}
+          onDelete={async () => {
+            // await deleteByteMutation({ variables: { spaceId: space.id, byteId: byteId! } });
+            setShowDeleteModal(false);
```
Comment (RobinNagpal) [inline academy-ui/src/components/shortVideos/View/Shorts.tsx:27]: @moezthedev  do we not need to update anything on the backend when we delete the short video?

Also, can you please remove the commented out code

Comment (moezthedev) [reply academy-ui/src/components/shortVideos/View/Shorts.tsx:27]: Yes we need to update backend and integrate it with frontend. Sami told me to make PR as He wanted to fix issues with three dots not able to click when I put them inside the card. Outside the card they were working as expected.

**Comment by MSamiTariq** [comment]
Closing this PR as its changes are included in this PR:
https://github.com/RobinNagpal/dodao-ui/pull/269

**Comment by MSamiTariq** [comment]
Closing this PR as its changes are included in this PR:
https://github.com/RobinNagpal/dodao-ui/pull/269

**Pull request closed without merging**